manager/src/main/java/org/openremote/manager/gateway/GatewayConnector.java

```diff
@@ -258,11 +259,12 @@ protected void getCapabilities() {
                     }
                 }
                 tunnellingSupported = response != null && response.isTunnelingSupported();
-                this.gatewayVersion = response.getVersion();
+                this.gatewayVersion = Objects.requireNonNull(response).getVersion();
```

Comment:
We're now throwing an exception here which is different behaviour, just check if `response != null` which is needed for older gateways that don't support the capabilities request